rest_log/hooks.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2021 Camptocamp SA (http://www.camptocamp.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+import logging
+
+_logger = logging.getLogger(__name__)
+
+
+def pre_init_hook(cr):
+    """On first install copy recods from shopfloor_log table if available.
+
+    """
+    cr.execute("SELECT 1 FROM pg_class WHERE relname = 'rest_log'")
+    if cr.fetchone():
+        # rest_log was already installed
+        return
+    cr.execute("SELECT 1 FROM pg_class WHERE relname = 'shopfloor_log'")
```

Comment:
?shopfloor?

Reply:
well, this module comes from there and I wanted to preserve old logs for shopfloor users. This is the only safe/sane way I've found w/out making it too complicated. Being just a query there's no tight dependency and I think is a good compromise.

Reply:
@simahawk Not a problem for me :smirk: